src/components/Examples.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 const builtApps = [
   {
     title: 'Roomie McRoomFace',
-    description: 'A room booking system for the UCL Engineering Hub. If you\'re an undergraduate student part of the engineering faculty, you can log in and start booking rooms.',
+    description: 'A room booking system for the UCL Engineering Hub. If you\'re an undergraduate student part of the engineering faculty, you can log in and start booking rooms. Fun fact: this was actually built before the API existed!',
```

Comment:
Doesn't this mean that we're lying then?

Reply:
@anushks

Reply:
minor tbh

Reply:
we could just remove it

Reply:
up to you tbh